### PR TITLE
fix scalar max bug

### PR DESCRIPTION
**Pull request by BBuf**
#close https://github.com/Oneflow-Inc/OneTeam/issues/2021

现在这个pr有一个问题，那就是我新增了一个Tensor (Tensor input, Scalar other) 的签名，这个签名和之前的"TensorTuple[values, indices] (Tensor input, Int32 dim, Bool keepdim=False) => Max"总是会匹配到放在前面的那个，导致这两种情况的例子无法同时运行成功。

图片

已解决

### CODE REVIEW

**Comment by hjchen2** [comment]
max不应该有scalar输入吧，下面的是pytorch max的用法。
```shell
 * (Tensor input)
 * (Tensor input, Tensor other, *, Tensor out)
 * (Tensor input, int dim, bool keepdim, *, tuple of Tensors out)
 * (Tensor input, name dim, bool keepdim, *, tuple of Tensors out)
```

**Comment by BBuf** [comment]
> max不应该有scalar输入吧，下面的是pytorch max的用法。
> 
> ```shell
>  * (Tensor input)
>  * (Tensor input, Tensor other, *, Tensor out)
>  * (Tensor input, int dim, bool keepdim, *, tuple of Tensors out)
>  * (Tensor input, name dim, bool keepdim, *, tuple of Tensors out)
> ```

pytorch是可以处理torch.max(torch.tensor(1), torch.tensor(2)) 这种case的

**Comment by hjchen2** [comment]
这个issue https://github.com/Oneflow-Inc/OneTeam/issues/2021 可以通过调整一下"TensorTuple[values, indices] (Tensor input, Int32 dim, Bool keepdim=False) => Max"和后面的“Tensor (Tensor input, Tensor other) => Max”的顺序来解决吧

**Comment by BBuf** [comment]
> 这个issue [Oneflow-Inc/OneTeam#2021](https://github.com/Oneflow-Inc/OneTeam/issues/2021) 可以通过调整一下"TensorTuple[values, indices] (Tensor input, Int32 dim, Bool keepdim=False) => Max"和后面的“Tensor (Tensor input, Tensor other) => Max”的顺序来解决吧

好像是的，我想多了，我改一下

**Review by hjchen2 — APPROVED** [review]

**Review by Flowingsun007 — APPROVED** [review]
